.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,772 @@
+Remove argument '{argument_docstring}' from the docstring""",
+                    )
+                )
+                bad_argument_function = True
+                break
+        if bad_argument_function:
+            break
+
+    # We found an error, no need to continue generating violations
+    if not bad_argument_function:
+        # Check whether docstring arguments match function arguments
+        for argument_docstring in arguments_docstring:
```

Ruff (0.8.2)

Loop control variable `argument_docstring` not used within loop body

(B007)